### ISSUE TITLE
Demotivational 

### ISSUE THREAD

**Issue by eboves** [tashsmit/Meme5#38]
Should be nice to have a preview while typing instead of having to press the previous button

**Comment by tashsmit** [comment]
I agree, its annoying to have to press the back button to rid the keyboard in order to preview the text. Maybe we can program the enter button to automatically preview.
